### DISCUSSION TITLE
AttributeError: 'Renderer' object has no attribute '_repr_mimebundle_'

### DISCUSSION THREAD

**Discussion by Sapta0802** [NVlabs/sionna#462] [Q&A]
hi, 

scene.preview() 

![image](https://github.com/NVlabs/sionna/assets/88283648/8169774c-a9c3-4c63-a70a-5bf58bc5321d)

thanks in advance.

**Comment by SebastianCa** [comment]
Hi @Sapta0802,

It seems like there is an issue with your Jupyter installation. Could you please provide more details about your setup (e.g., Jupyter version, etc.)? Does the problem occur for all tutorial notebooks with preview activated?

As a workaround, you could manually set `colab_compat = True` to run the notebooks.

**Reply by Sapta0802** [reply]
![image](https://github.com/NVlabs/sionna/assets/88283648/a2e70789-e05a-47f6-a4e8-1d0a4882a1aa)

![image](https://github.com/NVlabs/sionna/assets/88283648/7a3bfb7a-81d6-429a-9c2f-38a58f1176dd)

yes, the problem occurs for all tutorial notebooks with the preview activated. 

![image](https://github.com/NVlabs/sionna/assets/88283648/72933194-f864-48d8-b44d-656958e15fcb)
 this problem also occurs. I think, you are correct, I have downloaded some wrong versions, can you please help me which are correct?

**Reply by SebastianCa** [reply]
Please ensure that you have correctly installed all requirements (you can find them in requirements.txt). In particular, the following packages seem related to your problem: ``pythreejs >= 2.4.2``, ``ipywidgets >= 8.0.4``, ``ipydatawidgets == 4.3.2``, ``jupyterlab-widgets == 3.0.5``.

Remark: You might need to restart Jupyter after installing/upgrading these packages.
